### ISSUE TITLE
RADIUS auth

### ISSUE THREAD

**Issue by ghmj2417** [sipcapture/homer#242]
Using Homer 5.0.1, and I would like to implement RADIUS auth.  I found this link, [webHomer-authentication](https://github.com/sipcapture/homer/wiki/webHomer-authentication), however it appears to be out of date with an older version of Homer.  I looked at my current install and I found an api/Authentication directory with auth classes but none provide RADIUS support.  I also performed a search within the homer repo for anything related to RADIUS, and I found a few commits from 2012, [here](https://github.com/sipcapture/homer/commit/8bb6a44a4e68dc96f001659bc2819e073c6f143a). Was RADIUS auth removed?

Thank you for your help

**Comment by ghmj2417** [comment]
Please let me know if this isn't the proper place to ask about RADIUS.  I just figured this would be considered a potential issue since RADIUS was available and now it appears the auth class is missing.

**Comment by lmangani** [comment]
Sorry for the long queuing, we'll get to this issue eventually - we appreciate all reports plenty!

**Comment by lmangani** [comment]
btw @ghmj2417 webHomer is a super-obsoleted version, you should look into the current codebase

**Comment by ghmj2417** [comment]
@lmangani No worries, thank you for taking the time to respond.  We aren't using webHomer, we are using the 5.0.1 code so we are good there.  I just wanted to point out in my first post that I tried to do my diligence by researching the repo before I brought it up as an issue.  That is why I mentioned webHomer in my post.  I found that webHomer has the RADIUS auth class available, but the current code does not have the RADIUS auth class.
